plugins/modules/junos_l3_interfaces.py

```diff
@@ -133,47 +133,8 @@
     - name: ge-0/0/1
     - name: ge-0/0/2
   state: deleted
-
-# After state:
-# ------------
-#
-# admin# show interfaces
-# ge-0/0/1 {
-#     description "deleted L3 interface";
-# }
-# ge-0/0/2 {
-#     description "non L3 interface";
-#     unit 0 {
-#         family ethernet-switching {
-#             interface-mode access;
-#             vlan {
-#                 members 2;
-#             }
-#         }
-#     }
-# }
-
-
-# Using merged
-
-# Before state
-# ------------
-#
-# admin# show interfaces
-# ge-0/0/1 {
-#     description "L3 interface";
-#     unit 0 {
-#         family inet {
-#             address 10.200.16.10/24;
-#         }
-#     }
-# }
-# ge-0/0/2 {
-#     description "non configured interface";
-#     unit 0;
-# }
-
-- name: Merge provided configuration with device configuration (default operation is merge)
```

Comment:
Why are these lines deleted?

Reply:
It is a mistake, will update this. thanks